build/cli.ts

```diff
@@ -322,7 +325,7 @@ async function buildDocuments(
   );
   fs.writeFileSync(
     path.join(BUILD_OUT_ROOT, "allBrowserCompat.txt"),
```

Comment:
Not sure why I didn't do this when I wrote the code, but it would be nice to have a comment pointing out that this file is used in differy - it took me a bit of hunting to figure that out/remember